workers/tests/integrationtests/test_indicator_last_edit.py

```diff
@@ -30,18 +30,6 @@ def test(self):
         indicator.create_figure()
         self.assertIsNotNone(indicator.result.svg)
 
-    # TODO: Choose smaller test region for this test case
```

Comment:
Can not reproduce this test case with this regions.

Reply:
Meaning that for this region element count is bigger that contributions latest count

Reply:
Delete this for now.

Reply:
This test case has now an issue 
https://github.com/GIScience/ohsome-quality-analyst/issues/74